tests/components/gogogate2/test_init.py

```diff
@@ -92,5 +90,5 @@ async def test_auth_fail(hass: HomeAssistant) -> None:
     with patch(
         "homeassistant.components.gogogate2.get_data_update_coordinator",
         return_value=coordinator_mock,
-    ), pytest.raises(ConfigEntryNotReady):
```

Comment:
This test doesn't assert anything now. Why does it not raise anymore?

Reply:
This one is mocking the coordinator instead of the pypi. I'll fix it

Reply:
done